### PR TITLE
RAIN-2336 , RAIN-2489 (UI- Property Edit Property Application (Send Back to Citizen) flow)

### PR DESCRIPTION
**Pull request by vamshikrishnakole-wtt-egov**
RAIN-2336 , RAIN-2489 (UI- Property Edit Property Application (Send Back to Citizen) flow)

### CODE REVIEW

File: packages/modules/pt/src/pageComponents/PTWFApplicationTimeline.js [hunk 1]
```diff
@@ -84,6 +84,17 @@ const PTWFApplicationTimeline = (props) => {
             </Link>
           </div>
         );
+      case "REJECT":
```
Comment (jagankumar-egov) [inline packages/modules/pt/src/pageComponents/PTWFApplicationTimeline.js:87]: Update case edit itself ,

File: packages/modules/pt/src/pages/citizen/EditProperty/index.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+  // asessment details
+  data.PropertyType = data?.additionalDetails?.propertyType;
+  data.isResdential = data?.additionalDetails?.isResdential;
+  data.landarea = { floorarea: data?.landArea };
+
+  return data;
+}
+const EditProperty = ({ parentRoute }) => {
+  debugger;
```
Comment (jagankumar-egov) [inline packages/modules/pt/src/pages/citizen/EditProperty/index.js:70]: remove debuggers

File: packages/modules/pt/src/pageComponents/PTWFApplicationTimeline.js (hunk 1 above)
Comment (vamshikrishnakole-wtt-egov) [reply packages/modules/pt/src/pageComponents/PTWFApplicationTimeline.js]: resolved

File: packages/modules/pt/src/pages/citizen/EditProperty/index.js (hunk 2 above)
Comment (vamshikrishnakole-wtt-egov) [reply packages/modules/pt/src/pages/citizen/EditProperty/index.js]: removed

**Pull request merged**